app/assets/javascripts/components/tag_groups/ColorPicker.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import reactCSS from 'reactcss';
+import { TwitterPicker } from 'react-color';
+
+export default class ColorPick extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayColorPicker: false,
+      color: this.props.color
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+
+    this.handleClose = this.handleClick.bind(this);
+
+    this.handleChange = this.handleChange.bind(this);
+  };
+
+  handleClick(){
+    this.setState({ displayColorPicker: !this.state.displayColorPicker });
+  };
+
+  handleClose(){
+    this.setState({ displayColorPicker: false });
+  };
+
+  handleChange(color){
+    this.setState({ color: color.hex });
+  };
+
+  render() {
+    const styles = reactCSS({
```

Comment:
We should __not__ use it here, it's meant to use simple `style={{ backgroundColor: this.state.color }}` here on the tag we want this color. We shouldn't add a whole new dependency just because of a single dynamic style, I see no reason to add reactcss to the project right now.